services/src/configuration/src/main/java/org/openkilda/config/KafkaTopicsConfig.java

```diff
@@ -36,18 +36,28 @@
 
     @Key("flow")
     @FallbackKey("kafka.flow.topic")
-    @Default("kilda.flow")
+    @Default("kilda.flow.storm")
     String getFlowTopic();
 
+    @Key("flow.region")
+    @FallbackKey("kafka.flow.topic.region")
+    @Default("kilda.flow")
+    String getFlowRegionTopic();
+
     @Key("flow.status")
     @Default("kilda.flow.status")
     String getFlowStatusTopic();
 
     @Key("northbound")
     @FallbackKey("kafka.northbound.topic")
-    @Default("kilda.northbound")
+    @Default("kilda.northbound.storm")
     String getNorthboundTopic();
 
+    @Key("northbound.region")
+    @FallbackKey("kafka.northbound.topic.region")
+    @Default("kilda.northbound")
+    String getNorthboundRegionTopic();
+
     @Key("opentsdb")
     @Default("kilda.otsdb")
```

Comment:
Why you skipped this one for opentsdb storm topology? It would be great to have kafka topics for storm topologies written in a same way. Do you skipped this topic because we are not the only ones who read this topic?

Reply:
otsdb doesn't communicate with floodlight directly, we could change this also, but let's do it in a separate patch